.github/workflows/review-release-notes.yml

```diff
@@ -4,7 +4,6 @@ on: pull_request
 jobs:
   release_notes_review:
     runs-on: ubuntu-latest
```

Comment:
@kgal-pan - are we sure that we need to remove the - github.repository == 'demisto/content' condition? 
As I remember you are probably right, but have you tested it?

Reply:
Tested, details in https://github.com/demisto/content/pull/24061#issuecomment-1402200682.